source/How-To-Guides/Reproducing-CI-Results.rst

```diff
@@ -1,6 +1,27 @@
 Reproducing CI Results Locally
 ==============================
 
-TODO:
+The Space ROS project uses the `Earthly <https://earthly.dev>`_ utility to provide consistent builds between various infrastructure providers and local developer workstations.
+
+Space ROS CI runs on a daily basis using GitHub Actions and builds all packages included in the `Space ROS repos file <https://github.com/space-ros/space-ros/blob/main/ros2.repos>`_.
+Two artifacts are produced from these builds: an OCI-compliant container image published to Docker Hub as ``osrf/space-ros``, and a `build results archive <https://github.com/space-ros/process_sarif/blob/main/docs/build-results-archive.md>`_ available as a GitHub Actions artifact.
+
+Prerequisites
+-------------
+
+In order to reproduce these artifacts locally, you will need the following:
+
+* `earthly installation <https://earthly.dev/get-earthly>`__ which requires `Docker <https://docker.com>`__ (recommended) or `Podman <https://podman.io>`__.
```

Comment:
Even though it is an option, technically, should we call out Podman? Perhaps it is in the "possible, but unsupported" category.

Reply:
Removed in subsequent changes. Anyone who knows podman as a viable alternative likely doesn't need to be told as much in this tutorial.